www/js/diary/list.js

```diff
@@ -445,19 +448,23 @@ angular.module('emission.main.diary.list',['ui-leaflet',
    };
 
   $scope.choosePurpose = function() {
+    var isOther = false
     if($scope.chosen.purpose != "other_purpose"){
-      $scope.storePurpose($scope.chosen.purpose);
+      $scope.storePurpose($scope.chosen.purpose, isOther);
```

Comment:
why not just put `true` or `false` here directly instead of having an `isOther` variable?

Reply:
I added isOther variable for consistent temporary names to make it easy follow why it passes a boolean.